packages/channel-client/tests/fakes/fake-channel-provider.ts

```diff
@@ -263,9 +263,9 @@ export class FakeChannelProvider implements ChannelProviderInterface {
         break;
       // auto-close, if we received a close
       case 'closing':
-        this.latestState = {...this.latestState, turnNum, status: 'closed'};
-        this.notifyOpponent(this.latestState, 'ChannelUpdate');
-        this.notifyAppChannelUpdated(this.latestState);
+        this.setState({...this.latestState[params.data.channelId], turnNum, status: 'closed'});
+        this.notifyOpponent(this.latestState[params.data.channelId], 'ChannelUpdate');
+        this.notifyAppChannelUpdated(this.latestState[params.data.channelId]);
```

Comment:
Nitpick: Instead of repeating `params.data`, why don't we assign it to variable `const channel = params.data;` and use `channel` from then on.

Reply:
Yeah good shout

Reply:
ddd8c18